src/eddymotion/model/base.py

```diff
@@ -86,51 +94,18 @@ class BaseModel:
     __slots__ = (
         "_model",
         "_mask",
-        "_S0",
-        "_b_max",
         "_models",
```

Comment:
What is the difference between `model` and `models`? Unclear to me.

Reply:
According to an in-person conversation, we may need `models` to keep the model for each chunk of data that is processed. Keeping for now. Resolving.